src/main/resources/application.yml

```diff
@@ -88,6 +88,18 @@ accounts:
       api-host: ${ACCOUNTS_ROSTER_APIHOST:https://sandbox.safaricom.co.ke}
       pass-key: ${ACCOUNTS_ROSTER_PASSKEY:bfb279f9aa9bdbcf158e97dd71a467cd2e0c893059b10f78e6b72ada1ed2c919}
 
+paybill:
+  groups:
+    - business-short-code: 24322607
+      ams: "paygops"
+    - business-short-code: 123456
+      ams: "roster"
+    - business-short-code: "default"
+      ams: "paygops"
+
+currency: "USD"
+accountHoldingInstitutionId: "gorilla"
```

Comment:
Is this the default fall back value?

Reply:
No I have added the currency in groups block now.